core/Api/importCsvApi.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace ICT\Core\Api;
+
+/* * ***************************************************************
+ * Copyright © 2016 ICT Innovations Pakistan All Rights Reserved   *
+ * Developed By: Nasir Iqbal                                       *
+ * Website : http://www.ictinnovations.com/                        *
+ * Mail : nasir@ictinnovations.com                                 *
+ * *************************************************************** */
+
+use ICT\Core\Account;
+use ICT\Core\Api;
+use ICT\Core\CoreException;
+use ICT\Core\User;
+use ICT\Core\User\Permission;
+use SplFileInfo;
+
+class ImportCsvApi extends Api
+{
+
+
+
+ 
+  /**
+   * Import Users
+   *
+   * @url POST /import/users
+   * 
+   */
+  public function import_csv($data = array(), $mime = 'text/csv')
```

Comment:
Move this function into UserApi.php and drop this file.

Reply:
Moved to UserApi.php